### ISSUE TITLE
Linux evdev Wrong Anlog axis detected

### ISSUE THREAD

**Issue by EhartF** [RPCS3/rpcs3#5649]
System: 

- OS:
 OpenSUSE Tumbleweed
- Hardware:
 CPU: AMD FX8350
 GPU: AMD RX480
- RPCS3 0.0.6-7793-84d42ecb

Problem:
Configuration of Logitech F310 in Direct input mode.
The Gamepad gets detected, but initial configuration is completely wrong, the buttons are mapped wrong.
To get the buttons mapped correctly press them, select the button in the gui and release the button.
The analog sticks cannot be mapped correctly, some input on different sticks are read as inputs on the same axis. The log shows "! HLE: GetNextButtonPress: Evdev button LZ+ pressed with value 128 x2" where LZ+ is the axis, the value before the "x" (128) should be
 right-analoge(right), the value after the "x" (2) should be left-analoge(left).

The Gamped settings gui shows:
![f310-rpcs3](https://user-images.githubusercontent.com/47497748/52533961-7e7dba00-2d3b-11e9-9a4f-fea14a96ba3b.png)

**Comment by 13xforever** [comment]
Duplicate of #4754, or at least it's an issue with evdev rules for your device (like in #5580 and many other examples).

**Comment by Megamouse** [comment]
why is the wiki entry still nowhere to be found?

**Comment by greentop** [comment]
Setting the switch on the back of the controller to X should produce the following mapping as seen in the image below. (Ubuntu 18.04 & Logitech F310)

![screenshot from 2019-02-10 23-14-22](https://user-images.githubusercontent.com/2325766/52546212-5fa71400-2d8b-11e9-9252-2ba8128be0a6.png)

**Comment by Tatsh** [comment]
The fix is to do with flags in `.config/rpcs3/evdev_positive_axis.yml`. I have a PS2 controller adapter to USB and I had to set the following:

```yaml
ABS_X: true
ABS_Y: true
ABS_Z: true
```

Only after this did it stop seeing every button press as RZ+.

**Comment by Megamouse** [comment]
closing because a workaround is provided and this is a driver issue anyway.
